notifications/src/main/kotlin/notifications/service/EmailNotificationService.kt

```diff
@@ -0,0 +1,64 @@
+package notifications.service
+
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.flatMap
+import mu.KLogging
+import notifications.smtp.SMTPService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import provider.D3ClientProvider
+
+private const val NOTIFICATION_EMAIL = "noreply@d3.com"
+
+/**
+ * Service for email notifications
+ */
+@Component
+class EmailNotificationService(
+    @Autowired private val smtpService: SMTPService,
+    @Autowired private val d3ClientProvider: D3ClientProvider
+) : NotificationService {
+
+    override fun notifyDeposit(transferNotifyEvent: TransferNotifyEvent): Result<Unit, Exception> {
+        logger.info { "Notify deposit $transferNotifyEvent" }
+        val message =
+            "Hello ${transferNotifyEvent.accountId}. We are happy to notify that ${transferNotifyEvent.amount} ${transferNotifyEvent.assetName} has been deposited to your account."
```

Comment:
Hello **dear**
Are we really happy? 🤣 
I think it is better to ask @LiraLemur (https://github.com/LiraLemur) to help with that

Reply:
Is is the final state? I think something luke "Dear _. N tokens were deposited to your account." would be sufficient :) Though I am not sure about the context

Reply:
Oh, thank you @LiraLemur . Can you please also take a look at withdrawal message?